src/watchful/client.py

```diff
@@ -37,6 +37,7 @@
 HOST: str = "localhost"
 PORT: Optional[str] = "9002"
 API_TIMEOUT_SEC: int = 600
+TOKEN: Optional[str] = None
```

Comment:
I'm not really a fan of making this optional, but I get that this api leaves a lot to be desired.

Reply:
I don't like anything about this api, either